azure-functions/acbs-function/src/functions/acbs-amend-facility.js

```diff
@@ -138,23 +138,28 @@ df.app.orchestration('acbs-amend-facility', function* amendFacility(context) {
       amendments,
     });
 
+    /**
```

Comment:
are we happy for this to go back to main commented out and then be fixed as part of 7581?

Reply:
No we want this to be commented in `main` until next release which will enable it. Once an existing PR to merge `release-2.4.0` is merged to `main` we should not need to do anything more on `main`.